pom.xml

```diff
@@ -46,25 +50,14 @@
     </modules>
 
     <properties>
-        <jersey.version>2.19</jersey.version>
-        <confluent-common.version>3.3.0-SNAPSHOT</confluent-common.version>
-        <kafka.version>0.11.0.0-SNAPSHOT</kafka.version>
-        <kafka.scala.version>2.11</kafka.scala.version>
-        <log4j.version>1.7.6</log4j.version>
-        <restutils.version>3.3.0-SNAPSHOT</restutils.version>
+        <apache.directory.server.version>2.0.0-M22</apache.directory.server.version>
+        <apache.directory.api.version>1.0.0-M33</apache.directory.api.version>
         <avro.version>1.7.7</avro.version>
+        <confluent.maven.repo>http://packages.confluent.io/maven/</confluent.maven.repo>
+        <exec-maven-plugin.version>1.2.1</exec-maven-plugin.version>
         <jackson.version>2.5.4</jackson.version>
```

Comment:
Not sure what you mean about not using it -- the data in the _schemas topic uses Jackson to serialize it and we include a JsonSerializer/Deserializer in this repo...

In any case, my question was also considering that we might end up with dependencies on things like jackson in connectors which don't pull from rest-utils. I was trying to understand what our policy would be for where that version number should be placed -- all the way up in common or if it is placed in multiple projects where everything using rest-utils gets it from rest-utils and other things choose their own versions.